IfaceMgr/IfaceMgr.cpp

```diff
@@ -487,6 +487,40 @@ void TIfaceMgr::notifyScripts(const std::string& scriptName, SPtr<TMsg> question
     notifyScript(scriptName, action, params);
 }
 
+/* Notify script invoked when expiry occurs and lifetimes are reset.
+   This method can be modified as per the client requirements.
+*/
+void TIfaceMgr::notifyScripts(const std::string& scriptName, TNotifyScriptParams& params, TIAType type, char * addr, int len)
```

Comment:
You can tweak this one a bit, so it wouldn't call the script, just append addresses and prefixes to the params structure.

Reply:
I have corrected it.